packages/cli/src/commands/build.ts

```diff
@@ -54,6 +55,7 @@ import { importBuilders } from '../util/build/import-builders';
 import { initCorepack, cleanupCorepack } from '../util/build/corepack';
 import { sortBuilders } from '../util/build/sort-builders';
 import { toEnumerableError } from '../util/error';
+import { validateConfig } from '../util/dev/validate';
```

Comment:
Let's move this file?

Reply:
Done in [f838dfe](https://github.com/vercel/vercel/pull/8622/commits/f838dfe09772061357e6f1e5327be4b5cd961732)